src/oci.h

```diff
@@ -374,6 +416,9 @@ struct cc_oci_config {
 	 */
 	struct cc_oci_vm_cfg           *vm;
 
+	/** Network configuration. */
+	struct cc_oci_net_cfg           net;
```

Comment:
For consistency sake, do we want to carry a pointer to the network config here as well ?